packages/ui/react-ui-editor/src/extensions/markdown/decorate.ts

```diff
@@ -208,31 +211,35 @@ export const decorateMarkdown = (options: DecorateOptions = {}) => {
             update.docChanged ||
             update.viewportChanged ||
             update.focusChanged ||
-            update.transactions.some((tr) => tr.effects.some((e) => e.is(forceUpdate)))
+            update.transactions.some((tr) => tr.effects.some((e) => e.is(forceUpdate))) ||
+            (update.selectionSet && !options.selectionChangeDelay)
           ) {
             ({ deco: this.deco, atomicDeco: this.atomicDeco } = buildDecorations(
               update.view,
               options,
               update.view.hasFocus,
             ));
+
             this.clearUpdate();
           } else if (update.selectionSet) {
             this.scheduleUpdate(update.view);
           }
         }
 
-        clearUpdate() {
-          if (this.pendingUpdate > -1) {
-            window.clearTimeout(this.pendingUpdate);
-            this.pendingUpdate = -1;
-          }
-        }
-
+        // TODO(burdon): BUG: If the cursor is at the end of a link at the end of a line,
```

Comment:
https://github.com/dxos/dxos/assets/4529818/39b5e2b7-ebd5-440f-b3cd-9f2314959743

I captured a video of what I think you're describing here just yesterday

Reply:
i think marijn just fixed this (i merged it yesterday) -- can you check if you still see this in main.